main.py

```diff
@@ -64,10 +65,22 @@ def open_project(project_path: str):
         if os.path.isfile(stlpath):
             cntrl.load_stl(stlpath)
 
-        # try to open figures file
-        figpath = pathlib.Path(project_path, sett().slicing.splanes_file)
-        if os.path.isfile(figpath):
-            cntrl.load_planes(figpath)
+        if hasattr(sett().slicing, 'splanes_file'):
```

Comment:
Maybe it would be better to do the check in the version comparison method? If the project version is lower, than we additionally read the planes from the file and delete it